src/engine/Match_tainting_mode.ml

```diff
@@ -104,15 +104,6 @@ module DataflowY = Dataflow_core.Make (struct
   let short_string_of_node n = Display_IL.short_string_of_node_kind n.F2.n
 end)
 
-let convert_rule_id (id, _tok) =
```

Comment:
@brandonspark why did you remove this code?
@IagoAbal do you remember why we had this code in the first place?